src/vs/workbench/contrib/debug/browser/debugService.ts

```diff
@@ -432,7 +433,23 @@ export class DebugService implements IDebugService {
 							nls.localize('debugTypeMissing', "Missing property 'type' for the chosen launch configuration.");
 					}
 
-					await this.showError(message);
+					let actionList: IAction[] = [];
+					let action = {
+						id: 'Install Debug Extension',
```

Comment:
`id` by conventaion is only lowercase and does not have whitespaces

Reply:
Got it, makes sense. Adjusted to: `installAdditionalDebuggers` instead